<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info_Feed</class>
 <widget class="QWidget" name="Info_Feed">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Feed Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="QLTitle">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="EdtTitle"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="QLHomePage">
     <property name="text">
      <string>Home Page</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="EdtHomePage"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="QLLinK">
     <property name="text">
      <string>Feed URL</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QLineEdit" name="Edtlink"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="QLAuthor">
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="EdtAuthor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="QLFrequency">
     <property name="text">
      <string>Update Frequency</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="EdtUpdFrequency">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="QLUpdatePeriod">
     <property name="text">
      <string>Update Period</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="EdtUpdPeriod">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="QLDescirption">
     <property name="text">
      <string>Desciption</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="QLineEdit" name="EdtDescription"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="QLArticles">
     <property name="text">
      <string>Articles</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QTableView" name="FeedArticles">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashLine</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
